SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultWorkerJsonGenerator.cpp

```diff
@@ -46,12 +46,19 @@ bool GenerateAllDefaultWorkerJsons(bool& bOutRedeployRequired)
 
 	if (const USpatialGDKSettings* SpatialGDKSettings = GetDefault<USpatialGDKSettings>())
```

Comment:
I only wonder now how often this method is ran. If it's before running any kind of deployment, it should be ok.

Reply:
So, interesting thing about this, it's run before every *local* deployment.
There is no calling place for a cloud deployment. I believe this is actually an error even before this change, the worker.jsons just won't exist before the first local launch, so if you immediately try to cloud deploy, it will fail (probably). We probably haven't noticed this because almost everyone runs a local deployment first.

Reply:
Do you think I should address it in this PR? Or create a ticket?

Reply:
Nevermind, rewrote the entire thing.
Realized that these are basically user-written files, that we sometimes generate for them.
So switched to a model where files are not generated if not needed, and also stowed away if not needed.
Kind of got carried away...